<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamingView</class>
 <widget class="QDialog" name="StreamingView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QVideoWidget" name="m_preview" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="m_finishBtn">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>524</y>
     <width>130</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish Streaming</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelBtn">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>524</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel Streaming</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
